cmd/crio/main.go

```diff
@@ -290,7 +290,7 @@ func main() {
 			if err != nil {
 				return fmt.Errorf("invalid --cpu-profile value %q", err)
 			}
-			pprof.StartCPUProfile(f)
+			_ = pprof.StartCPUProfile(f)
```

Comment:
Actually I just noticed this is not hidden under the debug flag, we might not want to add this in normal execution as it may impact on performance I guess, I'll create a PR to enable this only when debug is set.